bigbluebutton-html5/imports/ui/components/common/Button/Button.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+const COLOR_TYPE = {
+  PRIMARY: 'primary',
+  SECONDARY: 'secondary',
+};
+
+const SIZE_VARIANT_TYPE = {
+  SMALL: 'sm',
+  MEDIUM: 'md',
+};
+
+const VARIANT_TYPE = {
+  CONATINED: 'contained',
+  OUTLINED: 'outlined',
+};
+
+const WEIGHT_VARIANT_TYPE = {
```

Comment:
should be `FONT_WEIGHT_TYPE`, similar to the variable name

Reply:
fixed